panel/layout.py

```diff
@@ -10,14 +10,14 @@
                            WidgetBox as BkWidgetBox, Spacer as BkSpacer)
 from bokeh.models.widgets import Tabs as BkTabs, Panel as BkPanel
 
-from .pane import Pane, PaneBase
+from .pane import create, Pane, PaneBase
```

Comment:
Can this import of Pane be deleted?  If not, seems like it should be marked `# noqa`.